backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/transformer/DocumentationUnitTransformer.java

```diff
@@ -67,14 +67,14 @@ public static DocumentationUnitDTO transformToDTO(
         currentDto.toBuilder()
             .id(updatedDomainObject.uuid())
             .documentNumber(updatedDomainObject.documentNumber())
-            .note(updatedDomainObject.note());
+            .note(StringUtils.normalizeSpace(updatedDomainObject.note()));
```

Comment:
Why did you normalize the note? I think it is a free text input like Kurz- and Langtexte and therefore shouldn't be changed, should it?

Reply:
Good point, I will change that back